lib/miq_automation_engine/engine/miq_ae_object.rb

```diff
@@ -116,7 +116,7 @@ def initialize(workspace, ns, klass, instance, object_name = nil)
 
       Benchmark.realtime_block(:inherit_time) do
         # Who do we inherit from
-        @inherits = @aec.inherits.blank? ? BASE_OBJECT : @aec.inherits
+        @inherits = BASE_OBJECT
```

Comment:
Does this imply that inheritance is expected to be completely removed?  Or only in one specific case?